src/Result/StoreRateSettings.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+declare(strict_types=1);
+
+namespace BTCPayServer\Result;
+
+use BTCPayServer\Util\PreciseNumber;
+
+class StoreRateSettings extends AbstractResult
+{
+    public function getSpread(): PreciseNumber
+    {
+        return PreciseNumber::parseFloat($this->getData()['spread']);
+    }
+
+    public function hasCustomScript(): bool
```

Comment:
`isCustomScript`

Reply:
you are not wrong but also did it deliberately here; maybe it is just me though but `hasCustomScript` makes imo more sense in that context; not happy with `effectiveScript` either, it should have been named `customScript` instead then all would make more sense imo; 

but yeah, not sure if worth it here

Reply:
Has is better too imo, just thought we were going with matching the field name. Let's stay with has.